src/services/teams/hooks/index.js

```diff
@@ -190,15 +190,16 @@ const blockedMethod = (hook) => {
  * @ifNotLocal
  * @param {Array of strings} keys
  * @param {Array::String || StringPath} path - the object path to filtered data in hook or 
+ * @param {Boolean} [_ifNotLocal=true] - pass all input to the ifNotLocal hook
  * @param {Object} [objectToFilter] - is optional otherwise the hook is used
  * @return {function::globalHooks.ifNotLocal(hook)}
  * @example filterToRelated(['_id','userIds'], 'result.data') - in hook.result.data all keys are removed that are not _id, or userIds
  */
-const filterToRelated = (keys, path, objectToFilter) => {
+const filterToRelated = (keys, path, _ifNotLocal = true, objectToFilter) => {
```

Comment:
is it safe to insert an optional parameter before the end? How commonly used is this function?

Reply:
I have write test for any case in filterToRelated.